docs/reference/api/security/TLS.md

```diff
@@ -41,6 +41,26 @@ Then create a file named `mbed_app.json` at the root of your application with th
 }
 ```
 
+### Mbed TLS platform context
+
+Some hardware accelerators require initialization, regardless of the specific cryptography engine. For this, we introduced `mbedtls_platform_setup()` and `mbedtls_platform_terminate()`.
+
+As the [examples](#mbed-tls-examples) show, you *must* call the `mbedtls_platform_setup()` function before you call any Mbed TLS API. After using the Mbed TLS API, you *must* call `mbedtls_platform_teardown()` to terminate the platform hardware driver. For readability reasons, we suggest you set the platform context parameter for these functions as `NULL`, as it is not being used in Mbed OS.
```

Comment:
I don't think it makes sense to say:
> we suggest you set the platform context parameter for these functions as `NULL`

This parameter must ALWAYS be NULL, and must be ignored by Mbed OS - because that's what the BLE code has to do.

It would be better to say:

> Mbed OS makes no use of the platform context parameter, and within Mbed OS functions should always be passed a `NULL` parameter for the context.

Reply:
Say that you MUST set the parameter to NULL implies that if it isn't NULL, then the function returns an error. That is not the case. The function simply ignores the parameter. This was don for backwards compatibility.
It is a suggestion to set to NULL, because it is ignored, and it is better readable that way, but it isn't mandatory